<!DOCTYPE html>
<html lang="en-US" dir="ltr">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width,initial-scale=1">
    <title>KohlbacherLab</title>
    <meta name="description" content="A VitePress site">
    <meta name="generator" content="VitePress v1.6.3">
    <link rel="preload stylesheet" href="/assets/style.B8Z8CkIc.css" as="style">
    <link rel="preload stylesheet" href="/vp-icons.css" as="style">
    
    <script type="module" src="/assets/app.Bz8AGn15.js"></script>
    <link rel="preload" href="/assets/inter-roman-latin.Di8DUHzh.woff2" as="font" type="font/woff2" crossorigin="">
    <link rel="modulepreload" href="/assets/chunks/theme.DcKVmJMn.js">
    <link rel="modulepreload" href="/assets/chunks/framework.BnfDpRyV.js">
    <link rel="modulepreload" href="/assets/publications_index.md.BpWB1jYs.lean.js">
    <script id="check-dark-mode">(()=>{const e=localStorage.getItem("vitepress-theme-appearance")||"auto",a=window.matchMedia("(prefers-color-scheme: dark)").matches;(!e||e==="auto"?a:e==="dark")&&document.documentElement.classList.add("dark")})();</script>
    <script id="check-mac-os">document.documentElement.classList.toggle("mac",/Mac|iPhone|iPod|iPad/i.test(navigator.platform));</script>
  </head>
  <body>
    <div id="app"><div class="Layout" data-v-5d98c3a5><!--[--><!--]--><!--[--><span tabindex="-1" data-v-0b0ada53></span><a href="#VPContent" class="VPSkipLink visually-hidden" data-v-0b0ada53>Skip to content</a><!--]--><!----><header class="VPNav" data-v-5d98c3a5 data-v-ae24b3ad><div class="VPNavBar" data-v-ae24b3ad data-v-6aa21345><div class="wrapper" data-v-6aa21345><div class="container" data-v-6aa21345><div class="title" data-v-6aa21345><div class="VPNavBarTitle" data-v-6aa21345 data-v-1168a8e4><a class="title" href="/" data-v-1168a8e4><!--[--><!--]--><!--[--><!--[--><!--[--><img class="VPImage dark logo" src="/images/icon/logo_light.png" alt data-v-8426fc1a><!--]--><!--[--><img class="VPImage light logo" src="/images/icon/logo_dark.png" alt data-v-8426fc1a><!--]--><!--]--><!--]--><!----><!--[--><!--]--></a></div></div><div class="content" data-v-6aa21345><div class="content-body" data-v-6aa21345><!--[--><!--]--><div class="VPNavBarSearch search" data-v-6aa21345><!--[--><!----><div id="local-search"><button type="button" class="DocSearch DocSearch-Button" aria-label="Search"><span class="DocSearch-Button-Container"><span class="vp-icon DocSearch-Search-Icon"></span><span class="DocSearch-Button-Placeholder">Search</span></span><span class="DocSearch-Button-Keys"><kbd class="DocSearch-Button-Key"></kbd><kbd class="DocSearch-Button-Key">K</kbd></span></button></div><!--]--></div><nav aria-labelledby="main-nav-aria-label" class="VPNavBarMenu menu" data-v-6aa21345 data-v-dc692963><span id="main-nav-aria-label" class="visually-hidden" data-v-dc692963> Main Navigation </span><!--[--><!--[--><a class="VPLink link VPNavBarMenuLink" href="/" tabindex="0" data-v-dc692963 data-v-e56f3d57><!--[--><span data-v-e56f3d57>Home</span><!--]--></a><!--]--><!--[--><a class="VPLink link VPNavBarMenuLink" href="/team.html" tabindex="0" data-v-dc692963 data-v-e56f3d57><!--[--><span data-v-e56f3d57>Team</span><!--]--></a><!--]--><!--[--><a class="VPLink link VPNavBarMenuLink" href="/research.html" tabindex="0" data-v-dc692963 data-v-e56f3d57><!--[--><span data-v-e56f3d57>Research</span><!--]--></a><!--]--><!--[--><a class="VPLink link vp-external-link-icon VPNavBarMenuLink" href="https://alma.uni-tuebingen.de/alma/pages/startFlow.xhtml?_flowId=searchCourseNonStaff-flow" target="_blank" rel="noreferrer" tabindex="0" data-v-dc692963 data-v-e56f3d57><!--[--><span data-v-e56f3d57>Teaching</span><!--]--></a><!--]--><!--[--><a class="VPLink link VPNavBarMenuLink" href="/publications.html" tabindex="0" data-v-dc692963 data-v-e56f3d57><!--[--><span data-v-e56f3d57>Publications</span><!--]--></a><!--]--><!--[--><a class="VPLink link VPNavBarMenuLink" href="/projects.html" tabindex="0" data-v-dc692963 data-v-e56f3d57><!--[--><span data-v-e56f3d57>Projects</span><!--]--></a><!--]--><!--[--><a class="VPLink link VPNavBarMenuLink" href="/software.html" tabindex="0" data-v-dc692963 data-v-e56f3d57><!--[--><span data-v-e56f3d57>Software</span><!--]--></a><!--]--><!--[--><a class="VPLink link VPNavBarMenuLink" href="/contact.html" tabindex="0" data-v-dc692963 data-v-e56f3d57><!--[--><span data-v-e56f3d57>Contact</span><!--]--></a><!--]--><!--]--></nav><!----><div class="VPNavBarAppearance appearance" data-v-6aa21345 data-v-6c893767><button class="VPSwitch VPSwitchAppearance" type="button" role="switch" title aria-checked="false" data-v-6c893767 data-v-5337faa4 data-v-1d5665e3><span class="check" data-v-1d5665e3><span class="icon" data-v-1d5665e3><!--[--><span class="vpi-sun sun" data-v-5337faa4></span><span class="vpi-moon moon" data-v-5337faa4></span><!--]--></span></span></button></div><div class="VPSocialLinks VPNavBarSocialLinks social-links" data-v-6aa21345 data-v-0394ad82 data-v-7bc22406><!--[--><!--]--></div><div class="VPFlyout VPNavBarExtra extra" data-v-6aa21345 data-v-bb2aa2f0 data-v-cf11d7a2><button type="button" class="button" aria-haspopup="true" aria-expanded="false" aria-label="extra navigation" data-v-cf11d7a2><span class="vpi-more-horizontal icon" data-v-cf11d7a2></span></button><div class="menu" data-v-cf11d7a2><div class="VPMenu" data-v-cf11d7a2 data-v-b98bc113><!----><!--[--><!--[--><!----><div class="group" data-v-bb2aa2f0><div class="item appearance" data-v-bb2aa2f0><p class="label" data-v-bb2aa2f0>Appearance</p><div class="appearance-action" data-v-bb2aa2f0><button class="VPSwitch VPSwitchAppearance" type="button" role="switch" title aria-checked="false" data-v-bb2aa2f0 data-v-5337faa4 data-v-1d5665e3><span class="check" data-v-1d5665e3><span class="icon" data-v-1d5665e3><!--[--><span class="vpi-sun sun" data-v-5337faa4></span><span class="vpi-moon moon" data-v-5337faa4></span><!--]--></span></span></button></div></div></div><div class="group" data-v-bb2aa2f0><div class="item social-links" data-v-bb2aa2f0><div class="VPSocialLinks social-links-list" data-v-bb2aa2f0 data-v-7bc22406><!--[--><!--]--></div></div></div><!--]--><!--]--></div></div></div><!--[--><!--]--><button type="button" class="VPNavBarHamburger hamburger" aria-label="mobile navigation" aria-expanded="false" aria-controls="VPNavScreen" data-v-6aa21345 data-v-e5dd9c1c><span class="container" data-v-e5dd9c1c><span class="top" data-v-e5dd9c1c></span><span class="middle" data-v-e5dd9c1c></span><span class="bottom" data-v-e5dd9c1c></span></span></button></div></div></div></div><div class="divider" data-v-6aa21345><div class="divider-line" data-v-6aa21345></div></div></div><!----></header><div class="VPLocalNav empty fixed" data-v-5d98c3a5 data-v-a6f0e41e><div class="container" data-v-a6f0e41e><!----><div class="VPLocalNavOutlineDropdown" style="--vp-vh:0px;" data-v-a6f0e41e data-v-8a42e2b4><button data-v-8a42e2b4>Return to top</button><!----></div></div></div><!----><div class="VPContent" id="VPContent" data-v-5d98c3a5 data-v-1428d186><div class="VPPage" data-v-1428d186><!--[--><!--]--><div style="position:relative;"><div><div class="container"><div class="page-title"><h1 class="page-title-text"><i class="fas fa-book"></i> Publications </h1></div><div class="d-flex flex-column gap-2"><div class="d-flex"><div class="ms-auto"><ul class="pagination vc-pagination"><!--[--><!--]--><!--[--><!--]--><!--[--><li class="page-item vc-pagination-item"><button class="active vc-pagination-link">1</button></li><li class="page-item vc-pagination-item"><button class="vc-pagination-link">2</button></li><!--]--><li class="page-item vc-pagination-item"><button class="vc-pagination-link"><i class="fa-solid fa-chevron-right"></i></button></li><li class="page-item vc-pagination-item"><button class="vc-pagination-link"><!--[--><i class="fa-solid fa-chevron-right"></i><i class="fa-solid fa-chevron-right"></i><!--]--></button></li></ul></div></div><div class="d-flex flex-column gap-3"><!--[--><div class="entity-card"><div class="d-flex flex-row algin-items-center"><div><h5><i class="fa-solid fa-book me-2"></i><!--[-->Statistical analysis of feature-based molecular networking results from non-targeted metabolomics data<!--]--></h5></div></div><hr><div class="d-flex flex-column gap-2"><div><h6>Author(s)</h6><div class="d-flex flex-row flex-wrap gap-1"><!--[--><div><!----> Abzer K. Pakkir Shah</div><div><!--[--> - <!--]--> Axel. Walter</div><div><!--[--> - <!--]--> Filip. Ottosson</div><div><!--[--> - <!--]--> Francesco. Russo</div><div><!--[--> - <!--]--> Marcelo. Navarro-Diaz</div><div><!--[--> - <!--]--> Judith. Boldt</div><div><!--[--> - <!--]--> Jarmo-Charles J. Kalinski</div><div><!--[--> - <!--]--> Eftychia Eva. Kontou</div><div><!--[--> - <!--]--> James. Elofson</div><div><!--[--> - <!--]--> Alexandros. Polyzois</div><div><!--[--> - <!--]--> . others</div><!--]--></div></div><div><h6>Abstract</h6><p class="mt-0">Feature-based molecular networking (FBMN) is a popular analysis approach for liquid chromatography–tandem mass spectrometry-based non-targeted metabolomics data. While processing liquid chromatography–tandem mass spectrometry data through FBMN is fairly streamlined, downstream data handling and statistical interrogation are often a key bottleneck. Especially users new to statistical analysis struggle to effectively handle and analyze complex data matrices. Here we provide a comprehensive guide for the statistical analysis of FBMN results, focusing on the downstream analysis of the FBMN output table. We explain the data structure and principles of data cleanup and normalization, as well as uni- and multivariate statistical analysis of FBMN results. We provide explanations and code in two scripting languages (R and Python) as well as the QIIME2 framework for all protocol steps, from data clean-up to statistical analysis. All code is shared in the form of Jupyter Notebooks (https://github.com/Functional-Metabolomics-Lab/FBMN-STATS). Additionally, the protocol is accompanied by a web application with a graphical user interface (https://fbmn-statsguide.gnps2.org/) to lower the barrier of entry for new users and for educational purposes. Finally, we also show users how to integrate their statistical results into the molecular network using the Cytoscape visualization tool. Throughout the protocol, we use a previously published environmental metabolomics dataset for demonstration purposes. Together, the protocol, code and web application provide a complete guide and toolbox for FBMN data integration, cleanup and advanced statistical analysis, enabling new users to uncover molecular insights from their non-targeted metabolomics data. Our protocol is tailored for the seamless analysis of FBMN results from Global Natural Products Social Molecular Networking and can be easily adapted to other mass spectrometry feature detection, annotation and networking tools.</p></div><div><span>Nature protocols, pp. 1–71, 2024</span></div></div></div><div class="entity-card"><div class="d-flex flex-row algin-items-center"><div><h5><i class="fa-solid fa-book me-2"></i><a class="" href="https://pubmed.ncbi.nlm.nih.gov/38909050" target="_blank"><!--[-->A study on interoperability between two Personal Ħealth Ŧrain infrastructures in leukodystrophy data analysis<!--]--></a></h5></div></div><hr><div class="d-flex flex-column gap-2"><div><h6>Author(s)</h6><div class="d-flex flex-row flex-wrap gap-1"><!--[--><div><!----> S. Welten</div><div><!--[--> - <!--]--> M. Arruda Botelho Herr</div><div><!--[--> - <!--]--> L. Hempel</div><div><!--[--> - <!--]--> D. Hieber</div><div><!--[--> - <!--]--> P. Placzek</div><div><!--[--> - <!--]--> M. Graf</div><div><!--[--> - <!--]--> S. Weber</div><div><!--[--> - <!--]--> L. Neumann</div><div><!--[--> - <!--]--> M. Jugl</div><div><!--[--> - <!--]--> L. Tirpitz</div><div><!--[--> - <!--]--> K. Kindermann</div><div><!--[--> - <!--]--> S. Geisler</div><div><!--[--> - <!--]--> L O Bonino. Silva Santos</div><div><!--[--> - <!--]--> S. Decker</div><div><!--[--> - <!--]--> N. Pfeifer</div><div><!--[--> - <!--]--> O. Kohlbacher</div><div><!--[--> - <!--]--> T. Kirsten</div><!--]--></div></div><!----><div><span>Sci Data, 11 (1), pp. 663, 2024</span></div></div></div><div class="entity-card"><div class="d-flex flex-row algin-items-center"><div><h5><i class="fa-solid fa-book me-2"></i><a class="" href="https://pubmed.ncbi.nlm.nih.gov/38867041" target="_blank"><!--[-->Ancient genomes reveal insights into ritual life at Chichén Itzá<!--]--></a></h5></div></div><hr><div class="d-flex flex-column gap-2"><div><h6>Author(s)</h6><div class="d-flex flex-row flex-wrap gap-1"><!--[--><div><!----> R. Barquera</div><div><!--[--> - <!--]--> O. vez</div><div><!--[--> - <!--]--> K. gele</div><div><!--[--> - <!--]--> P. rez-Ramallo</div><div><!--[--> - <!--]--> D I. ndez-Zaragoza</div><div><!--[--> - <!--]--> A. Szolek</div><div><!--[--> - <!--]--> A B. Rohrlach</div><div><!--[--> - <!--]--> P. Librado</div><div><!--[--> - <!--]--> A. Childebayeva</div><div><!--[--> - <!--]--> R A. Bianco</div><div><!--[--> - <!--]--> B S. Penman</div><div><!--[--> - <!--]--> V. a-Alonzo</div><div><!--[--> - <!--]--> M. Lucas</div><div><!--[--> - <!--]--> J C. Lara-Riegos</div><div><!--[--> - <!--]--> M E. Moo-Mezeta</div><div><!--[--> - <!--]--> J C. Torres-Romero</div><div><!--[--> - <!--]--> P. Roberts</div><div><!--[--> - <!--]--> O. Kohlbacher</div><div><!--[--> - <!--]--> C. Warinner</div><div><!--[--> - <!--]--> J. Krause</div><!--]--></div></div><!----><div><span>Nature, 2024</span></div></div></div><div class="entity-card"><div class="d-flex flex-row algin-items-center"><div><h5><i class="fa-solid fa-book me-2"></i><a class="" href="https://pubmed.ncbi.nlm.nih.gov/38449051" target="_blank"><!--[-->Prospective study validating a multidimensional treatment decision score predicting the 24-month outcome in untreated patients with clinically isolated syndrome and early relapsing-remitting multiple sclerosis, the ProVal-MS study<!--]--></a></h5></div></div><hr><div class="d-flex flex-column gap-2"><div><h6>Author(s)</h6><div class="d-flex flex-row flex-wrap gap-1"><!--[--><div><!----> A. Bayas</div><div><!--[--> - <!--]--> U. Mansmann</div><div><!--[--> - <!--]--> B I. n</div><div><!--[--> - <!--]--> V S. Hoffmann</div><div><!--[--> - <!--]--> A. Berthele</div><div><!--[--> - <!--]--> M. hlau</div><div><!--[--> - <!--]--> M C. Kowarik</div><div><!--[--> - <!--]--> M. Krumbholz</div><div><!--[--> - <!--]--> M. Senel</div><div><!--[--> - <!--]--> V. Steuerwald</div><div><!--[--> - <!--]--> M. Naumann</div><div><!--[--> - <!--]--> J. Hartberger</div><div><!--[--> - <!--]--> M. Kerschensteiner</div><div><!--[--> - <!--]--> E. Oswald</div><div><!--[--> - <!--]--> C. Ruschil</div><div><!--[--> - <!--]--> U. Ziemann</div><div><!--[--> - <!--]--> H. Tumani</div><div><!--[--> - <!--]--> I. Vardakas</div><div><!--[--> - <!--]--> F. Albashiti</div><div><!--[--> - <!--]--> F. Kramer</div><div><!--[--> - <!--]--> I. Soto-Rey</div><div><!--[--> - <!--]--> H. Spengler</div><div><!--[--> - <!--]--> G. Mayer</div><div><!--[--> - <!--]--> H A. Kestler</div><div><!--[--> - <!--]--> O. Kohlbacher</div><div><!--[--> - <!--]--> M. Hagedorn</div><div><!--[--> - <!--]--> M. Boeker</div><div><!--[--> - <!--]--> K. Kuhn</div><div><!--[--> - <!--]--> S. Buchka</div><div><!--[--> - <!--]--> F. Kohlmayer</div><div><!--[--> - <!--]--> J S. Kirschke</div><div><!--[--> - <!--]--> L. Behrens</div><div><!--[--> - <!--]--> H. Zimmermann</div><div><!--[--> - <!--]--> B. Bender</div><div><!--[--> - <!--]--> N. Sollmann</div><div><!--[--> - <!--]--> J. Havla</div><div><!--[--> - <!--]--> B. Hemmer</div><div><!--[--> - <!--]--> A. Berlis</div><div><!--[--> - <!--]--> B. Wiestler</div><div><!--[--> - <!--]--> T. mpfel</div><div><!--[--> - <!--]--> K. Seelos</div><div><!--[--> - <!--]--> J. nschede</div><div><!--[--> - <!--]--> R. Kemmner</div><div><!--[--> - <!--]--> M. Beer</div><div><!--[--> - <!--]--> J. Dietrich</div><div><!--[--> - <!--]--> J. Schaller</div><!--]--></div></div><div><h6>Abstract</h6><p class="mt-0">characteristics and (bio)markers that reliably predict the individual disease prognosis at disease onset are lacking. Cohort studies allow a close follow-up of MS histories and a thorough phenotyping of patients. Therefore, a multicenter cohort study was initiated to implement a wide spectrum of data and (bio)markers in newly diagnosed patients. months. Further objectives are refining the MS-TDS score and providing data to identify new markers reflecting disease course and severity. The project also provides a technical evaluation of the ProVal-MS cohort within the IT-infrastructure of the DIFUTURE consortium (Data Integration for Future Medicine) and assesses the efficacy of the data sharing techniques developed. Clinical cohorts provide the infrastructure to discover and to validate relevant disease-specific findings. A successful validation of the MS-TDS will add a new clinical decision tool to the armamentarium of practicing MS neurologists from which newly diagnosed MS patients may take advantage. Trial registration ProVal-MS has been registered in the German Clinical Trials Register, `Deutsches Register Klinischer Studien` (DRKS)-ID: DRKS00014034, date of registration: 21 December 2018; https://drks.de/search/en/trial/DRKS00014034.</p></div><div><span>Neurol Res Pract, 6 (1), pp. 15, 2024</span></div></div></div><div class="entity-card"><div class="d-flex flex-row algin-items-center"><div><h5><i class="fa-solid fa-book me-2"></i><a class="" href="https://pubmed.ncbi.nlm.nih.gov/38366242" target="_blank"><!--[-->OpenMS 3 enables reproducible analysis of large-scale mass spectrometry data<!--]--></a></h5></div></div><hr><div class="d-flex flex-column gap-2"><div><h6>Author(s)</h6><div class="d-flex flex-row flex-wrap gap-1"><!--[--><div><!----> J. Pfeuffer</div><div><!--[--> - <!--]--> C. Bielow</div><div><!--[--> - <!--]--> S. Wein</div><div><!--[--> - <!--]--> K. Jeong</div><div><!--[--> - <!--]--> E. Netz</div><div><!--[--> - <!--]--> A. Walter</div><div><!--[--> - <!--]--> O. Alka</div><div><!--[--> - <!--]--> L. Nilse</div><div><!--[--> - <!--]--> P D. Colaianni</div><div><!--[--> - <!--]--> D. McCloskey</div><div><!--[--> - <!--]--> J. Kim</div><div><!--[--> - <!--]--> G. Rosenberger</div><div><!--[--> - <!--]--> L. Bichmann</div><div><!--[--> - <!--]--> M. Walzer</div><div><!--[--> - <!--]--> J. Veit</div><div><!--[--> - <!--]--> B. Boudaud</div><div><!--[--> - <!--]--> M. Bernt</div><div><!--[--> - <!--]--> N. Patikas</div><div><!--[--> - <!--]--> M. Pilz</div><div><!--[--> - <!--]--> M P. Startek</div><div><!--[--> - <!--]--> S. Kutuzova</div><div><!--[--> - <!--]--> L. Heumos</div><div><!--[--> - <!--]--> J. Charkow</div><div><!--[--> - <!--]--> J C. Sing</div><div><!--[--> - <!--]--> A. Feroz</div><div><!--[--> - <!--]--> A. Siraj</div><div><!--[--> - <!--]--> H. Weisser</div><div><!--[--> - <!--]--> T M H. Dijkstra</div><div><!--[--> - <!--]--> Y. Perez-Riverol</div><div><!--[--> - <!--]--> H. st</div><div><!--[--> - <!--]--> O. Kohlbacher</div><div><!--[--> - <!--]--> T. Sachsenberg</div><!--]--></div></div><!----><div><span>Nat Methods, 2024</span></div></div></div><div class="entity-card"><div class="d-flex flex-row algin-items-center"><div><h5><i class="fa-solid fa-book me-2"></i><a class="" href="https://pubmed.ncbi.nlm.nih.gov/37997224" target="_blank"><!--[-->Precursor deconvolution error estimation: Ŧhe missing puzzle piece in false discovery rate in top-down proteomics<!--]--></a></h5></div></div><hr><div class="d-flex flex-column gap-2"><div><h6>Author(s)</h6><div class="d-flex flex-row flex-wrap gap-1"><!--[--><div><!----> K. Jeong</div><div><!--[--> - <!--]--> P T. Kaulich</div><div><!--[--> - <!--]--> W. Jung</div><div><!--[--> - <!--]--> J. Kim</div><div><!--[--> - <!--]--> A. Tholey</div><div><!--[--> - <!--]--> O. Kohlbacher</div><!--]--></div></div><div><h6>Abstract</h6><p class="mt-0">Top-down proteomics (TDP) directly analyzes intact proteins and thus provides more comprehensive qualitative and quantitative proteoform-level information than conventional bottom-up proteomics (BUP) that relies on digested peptides and protein inference. While significant advancements have been made in TDP in sample preparation, separation, instrumentation, and data analysis, reliable and reproducible data analysis still remains one of the major bottlenecks in TDP. A key step for robust data analysis is the establishment of an objective estimation of proteoform-level false discovery rate (FDR) in proteoform identification. The most widely used FDR estimation scheme is based on the target-decoy approach (TDA), which has primarily been established for BUP. We present evidence that the TDA-based FDR estimation may not work at the proteoform-level due to an overlooked factor, namely the erroneous deconvolution of precursor masses, which leads to incorrect FDR estimation. We argue that the conventional TDA-based FDR in proteoform identification is in fact protein-level FDR rather than proteoform-level FDR unless precursor deconvolution error rate is taken into account. To address this issue, we propose a formula to correct for proteoform-level FDR bias by combining TDA-based FDR and precursor deconvolution error rate.</p></div><div><span>Proteomics, 24 (3-4), pp. e2300068, 2024</span></div></div></div><div class="entity-card"><div class="d-flex flex-row algin-items-center"><div><h5><i class="fa-solid fa-book me-2"></i><a class="" href="https://pubmed.ncbi.nlm.nih.gov/37280276" target="_blank"><!--[-->The German Network for Personalized Medicine to enhance patient care and translational research<!--]--></a></h5></div></div><hr><div class="d-flex flex-column gap-2"><div><h6>Author(s)</h6><div class="d-flex flex-row flex-wrap gap-1"><!--[--><div><!----> A L. Illert</div><div><!--[--> - <!--]--> A. Stenzinger</div><div><!--[--> - <!--]--> M. Bitzer</div><div><!--[--> - <!--]--> P. Horak</div><div><!--[--> - <!--]--> V I. Gaidzik</div><div><!--[--> - <!--]--> Y. ller</div><div><!--[--> - <!--]--> J. Beha</div><div><!--[--> - <!--]--> Ö. ner</div><div><!--[--> - <!--]--> F. Schmitt</div><div><!--[--> - <!--]--> S. mann</div><div><!--[--> - <!--]--> S. Ossowski</div><div><!--[--> - <!--]--> C P. Schaaf</div><div><!--[--> - <!--]--> M. Hallek</div><div><!--[--> - <!--]--> T H. mmendorf</div><div><!--[--> - <!--]--> P. Albers</div><div><!--[--> - <!--]--> T. Fehm</div><div><!--[--> - <!--]--> P. Brossart</div><div><!--[--> - <!--]--> H. Glimm</div><div><!--[--> - <!--]--> D. Schadendorf</div><div><!--[--> - <!--]--> A. Bleckmann</div><div><!--[--> - <!--]--> C H. Brandts</div><div><!--[--> - <!--]--> I. Esposito</div><div><!--[--> - <!--]--> E. Mack</div><div><!--[--> - <!--]--> C. Peters</div><div><!--[--> - <!--]--> C. Bokemeyer</div><div><!--[--> - <!--]--> S. hling</div><div><!--[--> - <!--]--> T. Kindler</div><div><!--[--> - <!--]--> H. l</div><div><!--[--> - <!--]--> V. Heinemann</div><div><!--[--> - <!--]--> H. hner</div><div><!--[--> - <!--]--> R. Bargou</div><div><!--[--> - <!--]--> V. Ellenrieder</div><div><!--[--> - <!--]--> P. Hillemanns</div><div><!--[--> - <!--]--> F. Lordick</div><div><!--[--> - <!--]--> A. Hochhaus</div><div><!--[--> - <!--]--> M W. Beckmann</div><div><!--[--> - <!--]--> T. Pukrop</div><div><!--[--> - <!--]--> M. Trepel</div><div><!--[--> - <!--]--> L. Sundmacher</div><div><!--[--> - <!--]--> S. Wesselmann</div><div><!--[--> - <!--]--> G. Nettekoven</div><div><!--[--> - <!--]--> F. Kohlhuber</div><div><!--[--> - <!--]--> O. Heinze</div><div><!--[--> - <!--]--> J. Budczies</div><div><!--[--> - <!--]--> M. Werner</div><div><!--[--> - <!--]--> K. Nikolaou</div><div><!--[--> - <!--]--> A J. Beer</div><div><!--[--> - <!--]--> G. Tabatabai</div><div><!--[--> - <!--]--> W. Weichert</div><div><!--[--> - <!--]--> U. Keilholz</div><div><!--[--> - <!--]--> M. Boerries</div><div><!--[--> - <!--]--> O. Kohlbacher</div><div><!--[--> - <!--]--> J. Duyster</div><div><!--[--> - <!--]--> R. Thimme</div><div><!--[--> - <!--]--> T. Seufferlein</div><div><!--[--> - <!--]--> P. Schirmacher</div><div><!--[--> - <!--]--> N P. Malek</div><!--]--></div></div><!----><div><span>Nat Med, 2023</span></div></div></div><div class="entity-card"><div class="d-flex flex-row algin-items-center"><div><h5><i class="fa-solid fa-book me-2"></i><a class="" href="https://pubmed.ncbi.nlm.nih.gov/37173725" target="_blank"><!--[-->UmetaFlow: an untargeted metabolomics workflow for high-throughput data processing and analysis<!--]--></a></h5></div></div><hr><div class="d-flex flex-column gap-2"><div><h6>Author(s)</h6><div class="d-flex flex-row flex-wrap gap-1"><!--[--><div><!----> E E. Kontou</div><div><!--[--> - <!--]--> A. Walter</div><div><!--[--> - <!--]--> O. Alka</div><div><!--[--> - <!--]--> J. Pfeuffer</div><div><!--[--> - <!--]--> T. Sachsenberg</div><div><!--[--> - <!--]--> O S. Mohite</div><div><!--[--> - <!--]--> M. Nuhamunada</div><div><!--[--> - <!--]--> O. Kohlbacher</div><div><!--[--> - <!--]--> T. Weber</div><!--]--></div></div><div><h6>Abstract</h6><p class="mt-0">Metabolomics experiments generate highly complex datasets, which are time and work-intensive, sometimes even error-prone if inspected manually. Therefore, new methods for automated, fast, reproducible, and accurate data processing and dereplication are required. Here, we present UmetaFlow, a computational workflow for untargeted metabolomics that combines algorithms for data pre-processing, spectral matching, molecular formula and structural predictions, and an integration to the GNPS workflows Feature-Based Molecular Networking and Ion Identity Molecular Networking for downstream analysis. UmetaFlow is implemented as a Snakemake workflow, making it easy to use, scalable, and reproducible. For more interactive computing, visualization, as well as development, the workflow is also implemented in Jupyter notebooks using the Python programming language and a set of Python bindings to the OpenMS algorithms (pyOpenMS). Finally, UmetaFlow is also offered as a web-based Graphical User Interface for parameter optimization and processing of smaller-sized datasets. UmetaFlow was validated with in-house LC–MS/MS datasets of actinomycetes producing known secondary metabolites, as well as commercial standards, and it detected all expected features and accurately annotated 76</p></div><div><span>J Cheminform, 15 (1), pp. 52, 2023</span></div></div></div><div class="entity-card"><div class="d-flex flex-row algin-items-center"><div><h5><i class="fa-solid fa-book me-2"></i><a class="" href="https://pubmed.ncbi.nlm.nih.gov/36890446" target="_blank"><!--[-->The personalized cancer network explorer (PeCaX) as a visual analytics tool to support molecular tumor boards<!--]--></a></h5></div></div><hr><div class="d-flex flex-column gap-2"><div><h6>Author(s)</h6><div class="d-flex flex-row flex-wrap gap-1"><!--[--><div><!----> Mirjam. Figaschewski</div><div><!--[--> - <!--]--> Bilge. Sürü</div><div><!--[--> - <!--]--> Thorsten. Tiede</div><div><!--[--> - <!--]--> Oliver. Kohlbacher</div><!--]--></div></div><div><h6>Abstract</h6><p class="mt-0">Background: Personalized oncology represents a shift in cancer treatment from conventional methods to target specific therapies where the decisions are made based on the patient specific tumor profile. Selection of the optimal therapy relies on a complex interdisciplinary analysis and interpretation of these variants by experts in molecular tumor boards. With up to hundreds of somatic variants identified in a tumor, this process requires visual analytics tools to guide and accelerate the annotation process. Results: The Personal Cancer Network Explorer (PeCaX) is a visual analytics tool supporting the efficient annotation, navigation, and interpretation of somatic genomic variants through functional annotation, drug target annotation, and visual interpretation within the context of biological networks. Starting with somatic variants in a VCF file, PeCaX enables users to explore these variants through a web-based graphical user interface. The most protruding feature of PeCaX is the combination of clinical variant annotation and gene-drug networks with an interactive visualization. This reduces the time and effort the user needs to invest to get to a treatment suggestion and helps to generate new hypotheses. PeCaX is being provided as a platform-independent containerized software package for local or institution-wide deployment. PeCaX is available for download at https://github.com/KohlbacherLab/PeCaX-docker . Keywords: Clinical decision support; Gene drug interaction networks; Personalized oncology; Precision medicine.</p></div><div><span>BMC Bioinformatics, 24 (1), pp. 88, 2023</span></div></div></div><div class="entity-card"><div class="d-flex flex-row algin-items-center"><div><h5><i class="fa-solid fa-book me-2"></i><a class="" href="https://pubmed.ncbi.nlm.nih.gov/36823366" target="_blank"><!--[-->The HLA ligandome of oropharyngeal squamous cell carcinomas reveals shared tumour-exclusive peptides for semi-personalised vaccination<!--]--></a></h5></div></div><hr><div class="d-flex flex-column gap-2"><div><h6>Author(s)</h6><div class="d-flex flex-row flex-wrap gap-1"><!--[--><div><!----> L. Mühlenbruch</div><div><!--[--> - <!--]--> T. Abou-Kors</div><div><!--[--> - <!--]--> M L. Dubbelaar</div><div><!--[--> - <!--]--> L. Bichmann</div><div><!--[--> - <!--]--> O. Kohlbacher</div><div><!--[--> - <!--]--> M. Bens</div><div><!--[--> - <!--]--> J. Thomas</div><div><!--[--> - <!--]--> J. Ezić</div><div><!--[--> - <!--]--> J M. Kraus</div><div><!--[--> - <!--]--> H A. Kestler</div><div><!--[--> - <!--]--> A. Witzleben</div><div><!--[--> - <!--]--> J. Mytilineos</div><div><!--[--> - <!--]--> D. Fürst</div><div><!--[--> - <!--]--> D. Engelhardt</div><div><!--[--> - <!--]--> J. Doescher</div><div><!--[--> - <!--]--> J. Greve</div><div><!--[--> - <!--]--> P J. Schuler</div><div><!--[--> - <!--]--> M N. Theodoraki</div><div><!--[--> - <!--]--> C. Brunner</div><div><!--[--> - <!--]--> T K. Hoffmann</div><div><!--[--> - <!--]--> H G. Rammensee</div><div><!--[--> - <!--]--> J S. Walz</div><div><!--[--> - <!--]--> S. Laban</div><!--]--></div></div><div><h6>Abstract</h6><p class="mt-0">The immune peptidome of OPSCC has not previously been studied. Cancer-antigen specific vaccination may improve clinical outcome and efficacy of immune checkpoint inhibitors such as PD1/PD-L1 antibodies. 40) using immunoaffinity purification. The cohort included 22 HPV-positive (primarily HPV-16) and 18 HPV-negative samples. A benign reference dataset comprised of the HLA ligandomes of benign haematological and tissue datasets was used to identify tumour-associated antigens. MS analysis led to the identification of naturally HLA-presented peptides in OPSCC tumour tissue. In total, 22,769 peptides from 9485 source proteins were detected on HLA class I. For HLA class II, 15,203 peptides from 4634 source proteins were discovered. By comparative profiling against the benign HLA ligandomic datasets, 29 OPSCC-associated HLA class I ligands covering 11 different HLA allotypes and nine HLA class II ligands were selected to create a peptide warehouse. Tumour-associated peptides are HLA-presented on the cell surfaces of OPSCCs. The established warehouse of OPSCC-associated peptides can be used for downstream immunogenicity testing and peptide-based immunotherapy in (semi)personalised strategies.</p></div><div><span>Br J Cancer, pp. 1–11, 2023</span></div></div></div><!--]--></div><div class="d-flex"><div class="ms-auto"><ul class="pagination vc-pagination"><!--[--><!--]--><!--[--><!--]--><!--[--><li class="page-item vc-pagination-item"><button class="active vc-pagination-link">1</button></li><li class="page-item vc-pagination-item"><button class="vc-pagination-link">2</button></li><!--]--><li class="page-item vc-pagination-item"><button class="vc-pagination-link"><i class="fa-solid fa-chevron-right"></i></button></li><li class="page-item vc-pagination-item"><button class="vc-pagination-link"><!--[--><i class="fa-solid fa-chevron-right"></i><i class="fa-solid fa-chevron-right"></i><!--]--></button></li></ul></div></div><!--[--><h2>Errors</h2><div class="mt-1 mb-1"><!--[--><div class="alert alert-warning alert-sm">@DIAProt2021 could not be read. <div class="d-flex flex-column"><div><strong>Input</strong><br> Leon Bichmann, Shubham Gupta, George Rosenberger, Leon Kuchenbecker, Timo Sachsenberg, Phil Ewels, Oliver Alka, Julianus Pfeuffer, Oliver Kohlbacher, </div><div><strong>Trace</strong><br> @DIAProt2021: unexpected 10-part name &quot;Leon Bichmann, Shubham Gupta, George Rosenberger, Leon Kuchenbecker, Timo Sachsenberg, Phil Ewels, Oliver Alka, Julianus Pfeuffer, Oliver Kohlbacher, &quot;, dropping &quot;Leon Kuchenbecker, Timo Sachsenberg, Phil Ewels, Oliver Alka, Julianus Pfeuffer, Oliver Kohlbacher, &quot;</div></div></div><div class="alert alert-warning alert-sm">@Aschenbrenner2021-ld could not be read. <div class="d-flex flex-column"><div><strong>Input</strong><br> Aschenbrenner AC, Mouktaroudi M, Krämer B, Oestreich M, Antonakos N, Nuesch-Germano M, Gkizeli K, Bonaguro L, Reusch N, Baßler K, Saridaki M, Knoll R, Pecht T, Kapellos TS, Doulou S, Kröger C, Herbert M, Holsten L, Horne A, Gemünd ID, Rovina N, Agrawal S, Dahm K, van Uelft M, Drews A, Lenkeit L, Bruse N, Gerretsen J, Gierlich J, Becker M, Händler K, Kraut M, Theis H, Mengiste S, De Domenico E, Schulte-Schrepping J, Seep L, Raabe J, Hoffmeister C, ToVinh M, Keitel V, Rieke G, Talevi V, Skowasch D, Aziz NA, Pickkers P, van de Veerdonk FL, Netea MG, Schultze JL, Kox M, Breteler MMB, Nattermann J, Koutsoukou A, Giamarellos-Bourboulis EJ, Ulas T; German COVID-19 Omics Initiative (DeCOI)</div><div><strong>Trace</strong><br> @Aschenbrenner2021-ld: unexpected 55-part name &quot;Aschenbrenner AC, Mouktaroudi M, Krämer B, Oestreich M, Antonakos N, Nuesch-Germano M, Gkizeli K, Bonaguro L, Reusch N, Baßler K, Saridaki M, Knoll R, Pecht T, Kapellos TS, Doulou S, Kröger C, Herbert M, Holsten L, Horne A, Gemünd ID, Rovina N, Agrawal S, Dahm K, van Uelft M, Drews A, Lenkeit L, Bruse N, Gerretsen J, Gierlich J, Becker M, Händler K, Kraut M, Theis H, Mengiste S, De Domenico E, Schulte-Schrepping J, Seep L, Raabe J, Hoffmeister C, ToVinh M, Keitel V, Rieke G, Talevi V, Skowasch D, Aziz NA, Pickkers P, van de Veerdonk FL, Netea MG, Schultze JL, Kox M, Breteler MMB, Nattermann J, Koutsoukou A, Giamarellos-Bourboulis EJ, Ulas T; German COVID-19 Omics Initiative (DeCOI)&quot;, dropping &quot;Oestreich M, Antonakos N, Nuesch-Germano M, Gkizeli K, Bonaguro L, Reusch N, Baßler K, Saridaki M, Knoll R, Pecht T, Kapellos TS, Doulou S, Kröger C, Herbert M, Holsten L, Horne A, Gemünd ID, Rovina N, Agrawal S, Dahm K, van Uelft M, Drews A, Lenkeit L, Bruse N, Gerretsen J, Gierlich J, Becker M, Händler K, Kraut M, Theis H, Mengiste S, De Domenico E, Schulte-Schrepping J, Seep L, Raabe J, Hoffmeister C, ToVinh M, Keitel V, Rieke G, Talevi V, Skowasch D, Aziz NA, Pickkers P, van de Veerdonk FL, Netea MG, Schultze JL, Kox M, Breteler MMB, Nattermann J, Koutsoukou A, Giamarellos-Bourboulis EJ, Ulas T; German COVID-19 Omics Initiative (DeCOI)&quot;</div></div></div><div class="alert alert-warning alert-sm">@Klare2020 could not be read. <div class="d-flex flex-column"><div><strong>Input</strong><br> Klare, Juliane; Rurik, Marc; Rottmann, Eric; Bollen, Anke; Kohlbacher, Oliver; Fischer, Markus; Hackl, Thomas</div><div><strong>Trace</strong><br> @Klare2020: unexpected 8-part name &quot;Klare, Juliane; Rurik, Marc; Rottmann, Eric; Bollen, Anke; Kohlbacher, Oliver; Fischer, Markus; Hackl, Thomas&quot;, dropping &quot;Eric; Bollen, Anke; Kohlbacher, Oliver; Fischer, Markus; Hackl, Thomas&quot;</div></div></div><div class="alert alert-warning alert-sm">@Leitner_Structure_2020 could not be read. <div class="d-flex flex-column"><div><strong>Input</strong><br> Alexander Leitner, Alexandre M.J.J. Bonvin, Christoph H. Borchers, Robert J. Chalkley, Julia Chamot-Rooke, Colin W. Combe, Jürgen Cox, Meng-Qiu Dong, Lutz Fischer, Michael Götze, Fabio C. Gozzo, Albert J.R. Heck, Michael R. Hoopmann, Lan Huang, Yasushi Ishihama, Andrew R. Jones, Nir Kalisman, Oliver Kohlbacher, Karl Mechtler, Robert L. Moritz, Eugen Netz, Petr Novak, Evgeniy Petrotchenko, Andrej Sali, Richard A. Scheltema, Carla Schmidt, David Schriemer, Andrea Sinz, Frank Sobott, Florian Stengel, Konstantinos Thalassinos, Henning Urlaub, Rosa Viner, Juan A. Vizcaino, Marc R. Wilkins, Juri Rappsilber</div><div><strong>Trace</strong><br> @Leitner_Structure_2020: unexpected 36-part name &quot;Alexander Leitner, Alexandre M.J.J. Bonvin, Christoph H. Borchers, Robert J. Chalkley, Julia Chamot-Rooke, Colin W. Combe, Jürgen Cox, Meng-Qiu Dong, Lutz Fischer, Michael Götze, Fabio C. Gozzo, Albert J.R. Heck, Michael R. Hoopmann, Lan Huang, Yasushi Ishihama, Andrew R. Jones, Nir Kalisman, Oliver Kohlbacher, Karl Mechtler, Robert L. Moritz, Eugen Netz, Petr Novak, Evgeniy Petrotchenko, Andrej Sali, Richard A. Scheltema, Carla Schmidt, David Schriemer, Andrea Sinz, Frank Sobott, Florian Stengel, Konstantinos Thalassinos, Henning Urlaub, Rosa Viner, Juan A. Vizcaino, Marc R. Wilkins, Juri Rappsilber&quot;, dropping &quot;Robert J. Chalkley, Julia Chamot-Rooke, Colin W. Combe, Jürgen Cox, Meng-Qiu Dong, Lutz Fischer, Michael Götze, Fabio C. Gozzo, Albert J.R. Heck, Michael R. Hoopmann, Lan Huang, Yasushi Ishihama, Andrew R. Jones, Nir Kalisman, Oliver Kohlbacher, Karl Mechtler, Robert L. Moritz, Eugen Netz, Petr Novak, Evgeniy Petrotchenko, Andrej Sali, Richard A. Scheltema, Carla Schmidt, David Schriemer, Andrea Sinz, Frank Sobott, Florian Stengel, Konstantinos Thalassinos, Henning Urlaub, Rosa Viner, Juan A. Vizcaino, Marc R. Wilkins, Juri Rappsilber&quot;</div></div></div><!--]--></div><!--]--></div></div></div></div><!--[--><!--]--></div></div><footer class="VPFooter" data-v-5d98c3a5 data-v-e315a0ad><div class="container" data-v-e315a0ad><!----><p class="copyright" data-v-e315a0ad>Copyright © 2024-present KohlbacherLab</p></div></footer><!--[--><!--]--></div></div>
    <script>window.__VP_HASH_MAP__=JSON.parse("{\"contact.md\":\"Wmj-iMam\",\"index.md\":\"DiXqKWGy\",\"persons_baeuerle-famke.md\":\"DLU7F0yK\",\"persons_clin-lucien.md\":\"CVqHAopL\",\"persons_dijkstra-tjeerd.md\":\"k49m7p7G\",\"persons_elhabashy-hadeer.md\":\"gqa2N6a6\",\"persons_feroz-ayesha.md\":\"DJCCkdq3\",\"persons_herr-marius.md\":\"6gqrXKTD\",\"persons_hieber-david.md\":\"BB1zOkhJ\",\"persons_jeong-kyowon.md\":\"CcyEeSAV\",\"persons_jones-peter.md\":\"BbwS_KGn\",\"persons_kaipf-camill.md\":\"DJ5D_kkV\",\"persons_kohlbacher-oliver.md\":\"DyG1n-2r\",\"persons_kuchenbecker-leon.md\":\"RfMiThNY\",\"persons_leimeister-julia.md\":\"CK58YVk2\",\"persons_mueller-tom.md\":\"ClOeZKRC\",\"persons_nagel-andreas.md\":\"D6_kqNHC\",\"persons_pilz-matteo.md\":\"CCmsXewo\",\"persons_placzek-peter.md\":\"C7PFuQTd\",\"persons_roehl-alexander.md\":\"D8bzjZrp\",\"persons_sachsenberg-timo.md\":\"BIwR4dgC\",\"persons_siraj-arslan.md\":\"CrVkFDSw\",\"persons_sueruen-bilge.md\":\"Cu6Xs3px\",\"persons_sun-haoran.md\":\"DJJ7WcCy\",\"persons_walter-claudia.md\":\"B8i6VovU\",\"persons_wein-samuel.md\":\"DxF9KXCy\",\"projects_a4b.md\":\"C2XM88Fk\",\"projects_denbi.md\":\"Bx4dPhcs\",\"projects_difuture.md\":\"D88Ut2y2\",\"projects_epic-xs.md\":\"DjfF7cRk\",\"projects_food-profiling.md\":\"pwfFMGvK\",\"projects_index.md\":\"B3-bgqfk\",\"projects_pers-ons.md\":\"K3_nJm4e\",\"projects_privateaim.md\":\"B_i5rEqa\",\"publications_index.md\":\"BpWB1jYs\",\"research_index.md\":\"DN-fJeWd\",\"software_ball.md\":\"BnpXAErg\",\"software_epi-tool-kit.md\":\"6PjmQBY6\",\"software_fred-2.md\":\"DIAbD2ti\",\"software_index.md\":\"Cg5r3_b6\",\"software_multi-loc-2.md\":\"BOEAJZuo\",\"software_nrp-spredictor-2.md\":\"CdMyu-79\",\"software_open-ms.md\":\"DwcKUud9\",\"software_sher-loc-2.md\":\"wHypavRI\",\"software_xlec.md\":\"DV9Amifj\",\"software_y-loc.md\":\"B2m1lres\",\"teaching_index.md\":\"BF84Fxgn\",\"team_index.md\":\"D4mV5_mc\"}");window.__VP_SITE_DATA__=JSON.parse("{\"lang\":\"en-US\",\"dir\":\"ltr\",\"title\":\"KohlbacherLab\",\"description\":\"A VitePress site\",\"base\":\"/\",\"head\":[],\"router\":{\"prefetchLinks\":true},\"appearance\":true,\"themeConfig\":{\"search\":{\"provider\":\"local\"},\"logo\":{\"light\":\"/images/icon/logo_dark.png\",\"dark\":\"/images/icon/logo_light.png\"},\"socialLinks\":[],\"footer\":{\"copyright\":\"Copyright © 2024-present KohlbacherLab\"},\"editLink\":{\"pattern\":\"https://github.com/KohlbacherLab/homepage/edit/master/src/:path\",\"text\":\"Edit this page on GitHub\"},\"siteTitle\":false,\"nav\":[{\"text\":\"Home\",\"link\":\"/\"},{\"text\":\"Team\",\"link\":\"/team\",\"activeMatch\":\"/team\"},{\"text\":\"Research\",\"link\":\"/research\"},{\"text\":\"Teaching\",\"link\":\"https://alma.uni-tuebingen.de/alma/pages/startFlow.xhtml?_flowId=searchCourseNonStaff-flow\"},{\"text\":\"Publications\",\"link\":\"/publications\"},{\"text\":\"Projects\",\"link\":\"/projects\",\"activeMatch\":\"/projects/\"},{\"text\":\"Software\",\"link\":\"/software\",\"activeMatch\":\"/software/\"},{\"text\":\"Contact\",\"link\":\"/contact\"}],\"sidebar\":{\"/persons\":[{\"text\":\"Famke Bäuerle\",\"link\":\"/persons/baeuerle-famke\"},{\"text\":\"Lucien Clin\",\"link\":\"/persons/clin-lucien\"},{\"text\":\"Tjeerd Dijkstra\",\"link\":\"/persons/dijkstra-tjeerd\"},{\"text\":\"Hadeer Elhabashy\",\"link\":\"/persons/elhabashy-hadeer\"},{\"text\":\"Ayesha Feroz\",\"link\":\"/persons/feroz-ayesha\"},{\"text\":\"Dr. Marius Herr\",\"link\":\"/persons/herr-marius\"},{\"text\":\"David Hieber\",\"link\":\"/persons/hieber-david\"},{\"text\":\"Kyowon Jeong\",\"link\":\"/persons/jeong-kyowon\"},{\"text\":\"Peter J. Jones\",\"link\":\"/persons/jones-peter\"},{\"text\":\"Camill Kaipf\",\"link\":\"/persons/kaipf-camill\"},{\"text\":\"Oliver Kohlbacher\",\"link\":\"/persons/kohlbacher-oliver\"},{\"text\":\"Léon Kuchenbecker\",\"link\":\"/persons/kuchenbecker-leon\"},{\"text\":\"Julia Leimeister\",\"link\":\"/persons/leimeister-julia\"},{\"text\":\"Tom D. Müller\",\"link\":\"/persons/mueller-tom\"},{\"text\":\"Andreas Nagel\",\"link\":\"/persons/nagel-andreas\"},{\"text\":\"Matteo Pilz\",\"link\":\"/persons/pilz-matteo\"},{\"text\":\"Peter Placzek\",\"link\":\"/persons/placzek-peter\"},{\"text\":\"Alexander Röhl\",\"link\":\"/persons/roehl-alexander\"},{\"text\":\"Timo Sachsenberg\",\"link\":\"/persons/sachsenberg-timo\"},{\"text\":\"Arslan Siraj\",\"link\":\"/persons/siraj-arslan\"},{\"text\":\"Bilge Sueruen\",\"link\":\"/persons/sueruen-bilge\"},{\"text\":\"Haoran Sun\",\"link\":\"/persons/sun-haoran\"},{\"text\":\"Claudia Walter\",\"link\":\"/persons/walter-claudia\"},{\"text\":\"Samuel Wein\",\"link\":\"/persons/wein-samuel\"}],\"/software\":[{\"text\":\"Packages\",\"items\":[{\"text\":\"OpenMS\",\"link\":\"/software/open-ms\"},{\"text\":\"BALL\",\"link\":\"/software/ball\"},{\"text\":\"Fred2\",\"link\":\"/software/fred-2\"},{\"text\":\"XLEC\",\"link\":\"/software/xlec\"}]},{\"text\":\"Webserver\",\"items\":[{\"text\":\"EpiToolKit\",\"link\":\"/software/epi-tool-kit\"},{\"text\":\"NRPSpredictor2\",\"link\":\"/software/nrp-spredictor-2\"},{\"text\":\"MultiLoc2\",\"link\":\"/software/multi-loc-2\"},{\"text\":\"SherLoc2\",\"link\":\"/software/sher-loc-2\"},{\"text\":\"YLoc\",\"link\":\"/software/y-loc\"}]}],\"/projects\":[{\"text\":\"de.NBI\",\"link\":\"/projects/denbi\"},{\"text\":\"DIFUTURE\",\"link\":\"/projects/difuture\"},{\"text\":\"EPIC-XS\",\"link\":\"/projects/epic-xs\"},{\"text\":\"PersOns\",\"link\":\"/projects/pers-ons\"},{\"text\":\"PrivateAIM\",\"link\":\"/projects/privateaim\"},{\"text\":\"A4B\",\"link\":\"/projects/a4b\"},{\"text\":\"Food Profiling\",\"link\":\"/projects/food-profiling\"}]}},\"locales\":{},\"scrollOffset\":134,\"cleanUrls\":false}");</script>
    
  </body>
</html>